TEI.js

```diff
@@ -107,7 +107,7 @@ var ns = {"tei": "http://www.tei-c.org/ns/1.0",
 
 var exportedXMLIds = [];
 var generatedItems = [];
-var allItems = [];
+var allItems = {};
```

Comment:
Might as well fix it for the two arrays above (they should be objects). They're not used as arrays, so this does work